### PR TITLE
Increase field limit

### PR DESCRIPTION
**Pull request by ajdapretnar**
##### Issue

Attempt at solving #2404. 

##### Description of changes
Increase csv field limit.

##### Includes
- [X] Code changes
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by thocevar** [comment]
There seems to be a problem with sys.maxsize and underlying C types on some 64-bit systems (OverflowError: Python int too large to convert to C long). Consider replacing with 100*1024*1024. 100 MB should be more than enough. Besides, other components might choke on larger fields anyway.

**Comment by ajdapretnar** [comment]
So the argument should be 100*1024*1024 instead of sys?

**Comment by ajdapretnar** [comment]
Fixed.

File: Orange/data/io.py [hunk 1]
```diff
@@ -749,6 +749,10 @@ class CSVReader(FileFormat):
     SUPPORT_SPARSE_DATA = False
     PRIORITY = 20
 
+    def __init__(self, filename):
+        super().__init__(filename)
+        csv.field_size_limit(100*1024*1024)
```
Comment (kernc) [inline Orange/data/io.py]: This is a global side-effect. Put the line on top in a global namespace, somewhere shorty below the `csv` import. Precede by a comment:
> Support values longer than 128K (i.e. text contents features)

**Comment by ajdapretnar** [comment]
@kernc @thocevar I hope this is what you had in mind.

**Pull request merged**